tests/unit_tests/test_attachment.py

```
@@ -0,0 +1,1431 @@
+        with pytest.raises(ContentMismatchError, match="does not match declared MIME type"):
+            Attachment(
+                filename="archive.zip",
+                content=content,
+                mime_type="application/zip",
+            )
+
+    def test_content_mismatch_error_is_value_error(self):
+        """Test that ContentMismatchError is a subclass of ValueError."""
+        assert issubclass(ContentMismatchError, ValueError)
+        content = b"not a png file"
+        with pytest.raises(ValueError):
```

Ruff (0.14.13)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)